docs/en/operations/settings/settings.md

```diff
@@ -2855,4 +2855,47 @@ Possible values:
 
 Default value: `0`.
 
+## prefer_column_name_to_alias {#prefer-column-name-to-alias}
+
+Enables or disables using of the original column names instead of [aliases](../../sql-reference/syntax.md#syntax-expression_aliases). Enable this setting to make aliases syntax rules of the ClickHouse more compatible with most other database engines.
+
```

Comment:
```suggestion
Enables or disables using of the original column names instead of [aliases](../../sql-reference/syntax.md#syntax-expression_aliases). Enable this setting to make aliases syntax rules in ClickHouse more compatible with most other database engines.

```